src/controllers/tripsController.js

```diff
@@ -75,6 +78,7 @@ class tripsController {
         }
       });
     } catch (error) {
+      console.log(error);
```

Comment:
Better to push console.log free-code
```suggestion
      
      
```

Reply:
You mean an empty catch block?

Reply:
Nope. You can still pass res and req in this method so that you could be able to return meaningful message to the user